invenio_files_rest/storage.py

```diff
@@ -28,9 +28,14 @@
 
 import hashlib
 import time
+import tempfile
```

Comment:
Can you add these changes in a ``invenio_files_rest/contrib/s3.py`` and add ``both`` as an optional dependency in ``setup.py``.